src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -54,14 +55,15 @@ private void InstrumentModule()
                 var parameters = new ReaderParameters { ReadSymbols = true, AssemblyResolver = resolver };
                 ModuleDefinition module = ModuleDefinition.ReadModule(stream, parameters);
 
-                foreach (var type in module.GetTypes())
+                var moduleTypes = module.GetTypes().ToList();
```

Comment:
This will most likely add a performance overhead. `module.GetTypes()` uses `yield` to return and that makes it a perfect candidate to be used directly in a `foreach`. `moduleTypes` isn't used anywhere else, so it's actually an unnecessary allocation.

Reply:
Okay. I will fix it.